README.md

```diff
@@ -33,15 +33,7 @@ heavyai with GPU capabilities.
 To install heavyai for GPU Dataframe support (conda-only):
 
 ```bash
-conda create -n heavyai-gpu -c rapidsai -c nvidia -c conda-forge -c defaults python cudf cudatoolkit heavyai
-```
-
-Note that `pyheavydb` needs to be installed in the environment with pip
-until `heavydb` is available on conda-forge.
-
-```bash
-conda activate heavyai-gpu
-pip install pyheavydb
+conda create -n heavyai-gpu -c rapidsai -c nvidia -c conda-forge -c defaults python cudf cudatoolkit heavyai pyheavydb
```

Comment:
I suggest not including `cudatoolkit` package here if users have it installed in the system already. We can mention it as an alternative option though.

As in https://github.com/conda-forge/arrow-cpp-feedstock/issues/962, software packages should not depend on cudatoolkit as it is very heavy to install and it would make more sense to let users have it installed system-wide once rather than having multiple copies in different environments.

`python` dependency is superfluous and can be removed.

Finally, should we advocate using `mamba` as `conda` UX is exceptionally bad when installing these software.